docs/api-reference/data-management-api.md

```diff
@@ -59,10 +63,11 @@ This is a "soft delete" of the segment from Historicals.
 To undo this action, [mark the segment used](#mark-a-single-segment-as-used).
 
 Note that this endpoint returns an HTTP `200 OK` response code even if the segment ID or datasource doesn't exist.
+Check the response payload to confirm if any segment was actually updated.
 
 #### URL
 
-`DELETE` `/druid/coordinator/v1/datasources/{datasource}/segments/{segmentId}`
+`DELETE` `/druid/indexer/v1/datasources/{datasource}/segments/{segmentId}`
```

Comment:
I think it would also be helpful to include a brief note in the API docs mentioning that the coordinator-based segment management APIs have been deprecated in favor of the indexer-based ones. What do you think?

Reply:
It has been included in the info box at the top.